packages/plugins/src/plugins/eigenlayer/helper.ts

```diff
@@ -33,9 +35,19 @@ export async function getEigenLayerWithdrawals() {
   };
 
   const res: AxiosResponse<{ data: Withdrawal[] }> = await axios.get(
-    `${eigenlayerApiUrl}/withdrawals?status=queued_withdrawable&take=${eigenlayerApiWithdrawalsLimit}`,
+    `${eigenlayerApiUrl}/withdrawals?status=queued&take=${eigenlayerApiWithdrawalsLimit}`,
     options
   );
 
   return res.data;
 }
+
+/**
+ * Unwrap the backing layer token to the eigen token
+ * @param {Address} tokenAddress - The address of the token to unwrap
+ * @returns {Address} - The address of the unwrapped token
+ */
+export const unwrapBackingLayerToken = (tokenAddress: Address) => {
```

Comment:
```suggestion
export const unwrapBackingLayerToken = (tokenAddress: Address | string) => {
```

Feels like expanding the allowed input types (and properly handle it internally) decrease the forced-casts in the rest of the code ?

What do you think ?

Reply:
updated